src/Library/demos/Spin Button/main.json

```
@@ -0,0 +1,10 @@
+{
+  "name": "Spin Button",
+  "category": "user_interface",
+  "description": "Spin Button allows users to increment or decrement the displayed value of an attribute. ",
+  "panels": [
+    "code",
+    "preview"
```

Once we move the `SpinButton` to `main.blp`, the `ui` becomes more important.

```suggestion
    "ui",
    "preview"
```